theano/compile/function_module.py

```diff
@@ -121,26 +121,6 @@ def fgraph_updated_vars(fgraph, expanded_inputs):
     return updated_vars
 
 
-class Supervisor:
```

Comment:
Yes this is an option. I did see the pickle test failing and I was working on it yesterday. I thought maybe a cleaner option would be to unpickle the old test and remove the Supervisor from there and pickle it again. What do you think?